engine/samples/input/assets/sample.bind

```diff
@@ -6,26 +6,61 @@
             ],
             "gamepad": [
                 "RBumper"
+            ],
+            "mouse": [
+                "Left"
             ]
```

Comment:
I think you shouldn't add this here - I was a bit confused as to why the sample was closing when I was trying the mouse button showcase.